teamup-server/src/main/java/com/project/teamup/controller/UserController.java

```diff
@@ -44,7 +48,15 @@ public ResponseEntity generateToken(@PathVariable Long userId) {
         User userEntity = userService.findById(userId).get();
 
         userService.createVerificationTokenForUser(userEntity);
-        emailService.sendEmail(userEntity, EmailService.EmailType.FIRST_REGISTRATION);
+
```

Comment:
Sa mai tratezi si cazul in care utilizatorul nu are setata nicio limba in BD. Pune una default (eu am pus engleza) ca sa nu dea NullPointerException.